full-service/src/json_rpc/v1/e2e_tests/account/account_other.rs

```diff
@@ -184,527 +184,4 @@ mod e2e_account {
         );
         let _account = result.get("account").unwrap();
     }
-
-    #[test_with_logger]
-    fn test_e2e_get_balance(logger: Logger) {
```

Comment:
Why was all of this removed?

Reply:
Duplicate tests

Reply:
There are more dupe tests than this which I'll go through, didn't realize until I was testing the ones specific to this bugfix